bin/build/src/metabuild_common/aws.clj

```diff
@@ -20,6 +20,13 @@
             "--profile" aws-profile
             "cp" source dest))))
 
+(defn s3-copy-without-profile!
```

Comment:
could you explain why we need this version? why can't we use the existing on with profile?